cmd/tendermint/commands/rollback.go

```diff
@@ -0,0 +1,171 @@
+package commands
```

Comment:
I wonder if this should be part of the `tendermint` binary or if we should consider a new `tendermint-unsafe` tool that can have the existing unsafe commands plus this command? If possible I think it would be preferable not to crowd `tendermint` with commands like this. Could you start a new `cmd/tendermint-unsafe` and put this under there. We can add the other unsafe commands there in a later PR, and eventually deprecate them from the existing binary.

Also, I don't think the rollback parameters should be part of the main config file - they should just be flags on the command line.